src/web/modules/gateway_accounts/filter.njk

```diff
@@ -118,6 +119,16 @@
       }}
     </div>
 
+    <div class="govuk-grid-column-one-quarter inputs-less-margin">
+      {{ govukInput({
```

Comment:
Could I suggest that this gets added as a read-only filter? I think having the one entry point for this being from the search page is conceptually easier to describe to people. It also limits these in-line filters to simpler discrete filters (this change would set precedent for adding any inputs inline without considering how we'd best like 

It could use exactly the same data as here but potentially shown as a tag beneath the quick filters at the top, something similar to 
https://github.com/alphagov/pay-toolbox/blob/master/src/web/modules/transactions/list.njk#L47-L53

Reply:
@sfount OK I have pushed something now in 90bad05 not 100% sure about the placement but hopefully something like what you had in mind.
![Screenshot 2022-01-21 at 10 42 24](https://user-images.githubusercontent.com/1764158/150514021-939bf431-8225-4171-a06d-31a7f7766bb1.png)